### DISCUSSION TITLE
Create Es Index Failed, Show com.linecorp.armeria.client.ResponseTimeoutException: null

### DISCUSSION THREAD

**Discussion by luojie233** [apache/skywalking#9200] [Q&A] [answered]
skywalking version: 8.9.1 (9.0.0 same)
es version: 7.1.1

when start the service, it shows create index failed, this is the log:

![image](https://user-images.githubusercontent.com/17927730/173077283-ac973aa8-3311-4b12-9bdc-bf71aa99d4d8.png)

**Comment by luojie233** [comment, marked as answer]
It's a es setting problem.

**Reply by 00suren** [reply]
Hello, I encountered the same problem, what is the specific reason?

**Reply by luojie233** [reply]
I forgot which setting cause it, I just set es setting again，and restart it.
